recipe/meta.yaml

```
@@ -76,10 +76,11 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/bin/ncu                      # [linux]
-    - if not exist %LIBRARY_PREFIX%\nsight-compute exit 1  # [win]
+    - test -f $PREFIX/bin/ncu                                                  # [linux]
+    - test -f $PREFIX/bin/ncu-ui                                               # [linux64 or aarch64]
+    - if not exist %LIBRARY_PREFIX%\nsight-compute exit 1                      # [win]
     - if exist %LIBRARY_PREFIX%\nsight-compute\{{ version_split }}\lib exit 1  # [win]
-    - if not exist %PREFIX%\Scripts\ncu.bat exit 1  # [win]
+    - if not exist %PREFIX%\Scripts\ncu.bat exit 1                             # [win]
     - ncu --version
```

Would it make sense to test `ncu-ui` somehow?

```suggestion
    - ncu --version
    - ncu-ui --version                                                         # [aarch64]
```

---

It doesn't seem to expose command line options like version. I could do `--help-all`, but that might not be much better.

---

`--help-all` sounds good 👍

---

Looks like this is uncovering an issue with libGL missing.